internal/ethapi/api.go

```diff
@@ -816,7 +816,7 @@ func DoCall(ctx context.Context, b Backend, args CallArgs, blockNrOrHash rpc.Blo
 
 	var data []byte
 	if args.Data != nil {
-		data = []byte(*args.Data)
+		data = *args.Data
```

Comment:
Is this just a syntactical change or some real change? If the latter, I don't understand what difference it makes, if the former, that's fine.

Reply:
It's a syntactical change. IDE reminds me `Redundant type conversion`

Reply:
But anyway, I can revert this change. We have lots of `hexutil.Byte` to `byte` conversion. We can keep the consistency of the code.

Reply:
I'm fine with it, do as you please